auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Injection.kt

```diff
@@ -128,6 +130,7 @@ class Injection(
     val failedLoginCounter: Counter = meterRegistry.counter("login.failedLogins")
     val loginRateLimitCounter: Counter = meterRegistry.counter("login.rateLimitedRequests")
     val successfulLoginCounter: Counter = meterRegistry.counter("login.successfulLogins")
+    val ssoLoginCounter: Counter = meterRegistry.counter("login.ssoLogins")
```

Comment:
Does this need an alarm added to it like the others? Or is it just for reference?

Reply:
I think just for reference and to add to a dashboard at some point, it's Microsoft's job to worry about rate limiting for this.